lib/recipe/widgets/probe_list.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gibsonify/recipe/recipe.dart';
+
+class ProbeList extends StatelessWidget {
+  final int recipeIndex;
+
+  const ProbeList({Key? key, required this.recipeIndex}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
+      return ListView.builder(
+          padding: const EdgeInsets.all(2.0),
+          itemCount: state.recipes[recipeIndex].probes.length,
+          itemBuilder: (context, index) {
+            return Card(
+                child: InkWell(
```

Comment:
On desktop (Mac) or when using my keyboard (not the Android keyboard) on the emulator, I cannot type a space when editing Probe text, as this is registered as a tap. Not a huge deal now as we're targeting Android but probably wan't to deal with it in the future. I'm not sure if it is the `InkWell` doing this or it's something else.

Reply:
Okay, this is resolved now.